eos/references.yaml

```diff
@@ -289,6 +289,13 @@ BFvD:2016A:
     id: oai:arXiv.org:1608.07127
   inspire-id: Boer:2016iez
   title: QCD Factorization Theorem for $B \to \pi \pi \ell \nu$ Decays at Large Dipion Masses
+BGJvD:2019A:
+  authors: Bordone, M. and Gubernari, N. and van Dyk, D. and Jung, M.
+  eprint:
+    archive: arXiv
+    id: oai:arXiv.org:hep-ph/1912.09335
```

Comment:
This is incorrect I think. Older entries are ``hep-ph/YYMMABC``. Newer ones are simply ``YYMM.ABCDE``.

Reply:
Ah sure, changed it.